package.json

```diff
@@ -122,57 +123,51 @@
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^29.4.0",
     "@types/lodash.debounce": "^4.0.7",
-    "@types/node": "^17.0.21",
+    "@types/node": "^18.11.18",
     "@types/react": "^17.0.53",
     "@types/react-beautiful-dnd": "^13.1.3",
     "@types/react-dom": "^17.0.18",
     "@types/react-portal": "^4.0.4",
     "@types/svg4everybody": "^2.1.2",
-    "axe-core": "^4.4.3",
+    "axe-core": "4.4.3",
```

Comment:
Locking axe-core to 4.4.3. There's timeouts on newer versions per 4862f030fb8340221d03661179cbbf04cfaa9525